classes/notification/managerDelegate/ConfigurationNotificationManager.inc.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @file classes/notification/managerDelegate/ConfigurationNotificationManager.inc.php
```

Comment:
Was this taken from a 3.3.0-x class as a template? This should be renamed to `.php` and moved into a namespace; have a look at the other notification manager classes in the `main` branch as an example.

Reply:
There should be code auto-formatting tools in the codebase that take care of things like indentation; it looks like those aren't set up in your checkout. For pkp-lib, it's a tool called "Captain Hook" that runs PHP CS Fixer on a call to `git commit`. Can you check that toolset to make sure it's working? (It should e.g. convert tabs to spaces, which we've adopted in `main` per the [PSR](https://www.php-fig.org/psr/psr-2/).)